packages/lesswrong/components/vulcan-forms/Form.tsx

```diff
@@ -36,8 +36,7 @@ type FormFieldUnfinished<N extends CollectionNameString> = Partial<FormField<N>>
 // props that should trigger a form reset
 const RESET_PROPS = [
   'collection', 'collectionName', 'typeName', 'document', 'schema', 'currentUser',
-  'fields', 'removeFields',
-  'prefilledProps' // TODO: prefilledProps should be merged instead?
+  'fields', 'removeFields', 'prefilledProps'
```

Comment:
Perhaps we should update the comment above to note that `prefilledProps` functions differently?